action.yml

```diff
@@ -59,9 +59,11 @@ runs:
       export UNESCAPED_BRANCH_NAME=${{ inputs.branch-name }}
       
       branch=$(echo "$UNESCAPED_BRANCH_NAME" | sed "s%refs\/heads\/%%g")
-      clean_branch=$(echo "$branch" | sed "s/_//g" | tr "[:upper:]" "[:lower:]" | cut -c -38)
+      cdk_branch=$(echo "$branch" | sed "s/_//g" | cut -c -38)
+      clean_branch=$(echo "$branch" tr "[:upper:]" "[:lower:]" )
```

Comment:
I don't think the current `clean_branch` needs to change at all. Also I think this is producing unexpected values:

```shell
clean_branch=$(echo "$branch" tr "[:upper:]" "[:lower:]" )                                 
echo $clean_branch
test_it-out_123 tr [:upper:] [:lower:]
```

Reply:
You're missing a pipe operator

```shell
clean_branch=$(echo "$branch" | tr "[:upper:]" "[:lower:]" )                               
echo $clean_branch                                          
test_it-out_123
```

But I think we should still have the `sed "s/_//g"` as well. In that case, the current stuff should stay the same and we should just pass `$clean_branch` to the cdk stack name